scipy/special/_testutils.py

```diff
@@ -174,6 +174,7 @@ def get_tolerances(self, dtype):
 
     def check(self, data=None, dtype=None, dtypes=None):
         """Check the special function against the data."""
+        __tracebackhide__ = True
```

Comment:
Better to silence the specific error:
`__tracebackhide__ = operator.methodcaller('errisinstance', AssertionError)`
in case there's a bug here.

Reply:
Good point; done.